cmd/cored/main.go

```diff
@@ -140,11 +136,11 @@ func main() {
 	if err != nil {
 		chainlog.Fatalkv(ctx, chainlog.KeyError, err)
 	}
-	listener, cert, err := maybeUseTLS(listener)
+	listener, tlsConfig, err := maybeUseTLS(listener)
```

Comment:
ah ok, wdyt about updating the `TLSConfig` struct comment to explain that?

The mention of server tripped me up:
```
// TLSConfig returns a TLS config suitable for use
// as a Chain Core client and server.
```